### DISCUSSION TITLE
Add short descriptions to setting options

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#662] [Q&A] [answered]
Add short descriptions (maximum one sentence) to the setting options. This could/should be a community task.

**Comment by devemux86** [comment, marked as answer]
@printmaps Thanks for the suggestion.

Most settings already have descriptions.

If you find some that need better titles or descriptions, we can improve them.

Cruiser supports too many languages, so this is not an easy task.
If only English were available, this would be much easier.

**Comment by 0709wiwiwi** [comment]
https://github.com/devemux86/cruiser/wiki/How-to%3F
See item: How to display usefull Cruiser app settings hints.

**Comment by printmaps** [comment]
What I mean is something like this (option + short description):

![option-with-description](https://github.com/user-attachments/assets/01ee39ef-3ff0-4a19-9489-ed1321cf00d2)
